lang/udf_cache.hh

```diff
@@ -0,0 +1,154 @@
+/*
+ */
+
```

Comment:
?

Reply:
Seems like I've copied a corrupted copyright clause (I see it appears in other files too), but I'll fix it here